langlib/lang.int/src/main/ballerina/int.bal

```diff
@@ -123,6 +160,12 @@ public isolated function min(int n, int... ns) returns int = @java:Method {
 # The first character may be `+` or `-`.
 # This is the inverse of function ``value:toString`` applied to an `int`.
 #
+# ```ballerina
+# int:fromString("76") ⇒ 76
+# 
+# int:fromString("0xFFFF") ⇒ error
+# ```
```

Comment:
shall we add an example for a minus value? Ex: `int:fromString("-120") ⇒ -120`

Reply:
Added in https://github.com/ballerina-platform/ballerina-lang/pull/38917/commits/bef81eacf094807ffddb1ce9a5eb594b4b1b8bee